inc/options/minify.php

```diff
@@ -1000,24 +1000,52 @@
 		<table class="form-table">
 			<?php
 			$c           = Dispatcher::config();
+			$disabled    = ! $c->get_boolean( 'minify.enabled' );
 			$wp_disabled = ! $c->get_boolean( 'minify.wp_cron' );
 
+			if ( $disabled ) {
+				echo wp_kses(
+					sprintf(
+						// Translators: 1 opening HTML div tag followed by opening HTML p tag, 2 opening HTML a tag,
+						// Translators: 3 closing HTML a tag, 4 closing HTML p tag followed by closing HTML div tag.
+						__( '%1$sMinify is disabled! Enable it %2$shere%3$s to enable this feature.%4$s', 'w3-total-cache' ),
+						'<div class="notice notice-error inline"><p>',
+						'<a href="' . esc_url( admin_url( 'admin.php?page=w3tc_general#minify' ) ) . '">',
+						'</a>',
+						'</p></div>'
+					),
+					array(
+						'div' => array(
+							'class' => array(),
+						),
+						'p'   => array(),
+						'a'   => array(
+							'href' => array(),
+						),
+					)
+				);
+			}
+
 			Util_Ui::config_item(
 				array(
 					'key'            => 'minify.wp_cron',
 					'label'          => esc_html__( 'Enable WP-Cron Event', 'w3-total-cache' ),
 					'checkbox_label' => esc_html__( 'Enable', 'w3-total-cache' ),
 					'control'        => 'checkbox',
 					'description'    => esc_html__( 'Enabling this will schedule a WP-Cron event that will flush the Minify Cache. If you prefer to use a system cron job instead of WP-Cron, you can schedule the following command to run at your desired interval: "wp w3tc flush minify".', 'w3-total-cache' ),
+					'disabled'       => $disabled,
 				)
 			);
 
 			$time_options = array();
+			$timezone     = new \DateTimeZone( get_user_meta( get_current_user_id(), 'timezone', true ) ?: wp_timezone()->getName() );
+
 			for ( $hour = 0; $hour < 24; $hour++ ) {
-				foreach ( array( '00', '30' ) as $minute ) {
-					$time_value                  = $hour * 60 + intval( $minute );
-					$time_label                  = gmdate( 'g:i a', strtotime( sprintf( '%02d:%s', $hour, $minute ) ) );
-					$time_options[ $time_value ] = $time_label;
+				foreach ( array('00', '30') as $minute ) {
+					$time_value                = $hour * 60 + intval( $minute );
+					$scheduled_time            = new \DateTime( "{$hour}:{$minute}", $timezone );
```

Comment:
If the site's timezone is only used here, use `wp_timezone()` instead of the variable above.

Reply:
Updated